app/components/views/GetStartedPage/CreateWallet/CreateWalletForm/ExistingSeed/Form.js

```diff
@@ -3,132 +3,66 @@ import SeedHexEntry from "./SeedHexEntry";
-            toggleAction={this.handleToggle}
-          />
-        </div>
-        <div className="confirm-seed-row seed">
-          <div className="confirm-seed-label-text seed">
-            <ConfirmSeedMsg />
-          </div>
-          {seedType == "words" && Array.isArray(seedWords) ?
-            <div className="seedArea">
-              {this.state.showPasteWarning &&
-              <div className="orange-warning">
-                <T id="confirmSeed.warnings.pasteExistingSeed" m="*Please make sure you also have a physical, written down copy of your seed." />
-              </div>}
-              {this.state.showPasteError &&
-              <div className="seedError">
-                <T id="confirmSeed.warnings.pasteExistingError" m="* Please paste a valid 33 word seed."/>
-              </div>}
-              {seedWords.map((seedWord) => {
-                const className = seedWord.word ? seedWord.error ? "seedWord error" : "seedWord populated" : "seedWord restore";
-                return (
-                  <SingleSeedWordEntry
-                    className={className}
-                    onChange={onChangeSeedWord}
-                    onPaste={this.handleOnPaste}
-                    seedWord={seedWord}
-                    value={{ name: seedWord.word }}
-                    key={seedWord.index}
-                    onPasteFromClipboard={this.pasteFromClipboard}
-                  />);
-              })}
-            </div> :
-            <div className="seedArea hex">
-              <SeedHexEntry
-                onChange={setSeedWords}
-              />
+        {seedType == "words" && Array.isArray(seedWords) ?
+          <div className="seedArea">
+            {showPasteWarning &&
+            <div className="orange-warning">
+              <T id="confirmSeed.warnings.pasteExistingSeed" m="*Please make sure you also have a physical, written down copy of your seed." />
+            </div>}
+            {showPasteError &&
+            <div className="seedError">
+              <T id="confirmSeed.warnings.pasteExistingError" m="* Please paste a valid 33 word seed."/>
             </div>}
-          <div className="input-form-error">
-            {errors.length > 0 && <div>{errors}</div>}
-          </div>
+            {seedWords.map((seedWord) => {
+              const className = seedWord.word ? seedWord.error ? "seedWord error" : "seedWord populated" : "seedWord restore";
+              return (
+                <SingleSeedWordEntry
+                  className={className}
+                  onChange={onChangeSeedWord}
+                  onPaste={handleOnPaste}
+                  seedWord={seedWord}
+                  value={{ name: seedWord.word }}
+                  key={seedWord.index}
+                  onPasteFromClipboard={pasteFromClipboard}
+                />);
+            })}
+          </div> :
+          <div className="seedArea hex">
+            <SeedHexEntry
+            // needs implementing
```

Comment:
Is this still applicable?

Reply:
Yeah, that is expected.

Right now on master, the method used to handle hex seed paste is `setSeedWords`, which is a copy of `onChangeSeedWord`. But the problem is pasting is not working on master, I believe because of handlePaste method, so I removed this method and I will work on this next.

But if you prefer, I can add into this PR. The reason I did not add into it, it is only to not have a big diff and fix 1 issue with 1 PR.

Reply:
Ah, ok. I guess it's fine if it's on another PR.